pex/interpreter_constraints.py

```diff
@@ -317,12 +317,11 @@ def iter_compatible_versions(
     PythonVersion(Lifecycle.EOL, 3, 5, 10),
     PythonVersion(Lifecycle.EOL, 3, 6, 15),
     PythonVersion(Lifecycle.EOL, 3, 7, 17),
-    # ^-- EOL --^
     PythonVersion(Lifecycle.STABLE, 3, 8, 18),
     PythonVersion(Lifecycle.STABLE, 3, 9, 18),
     PythonVersion(Lifecycle.STABLE, 3, 10, 13),
-    PythonVersion(Lifecycle.STABLE, 3, 11, 7),
-    PythonVersion(Lifecycle.STABLE, 3, 12, 1),
+    PythonVersion(Lifecycle.STABLE, 3, 11, 8),
+    PythonVersion(Lifecycle.STABLE, 3, 12, 2),
```

Comment:
I'm assuming this is tangentially-related general bumping because these have also been recently released?

Reply:
Yes. I should have done it in #2360.